<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssayShiledDlg</class>
 <widget class="BaseDlg" name="AssayShiledDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1225</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">AssayShiledDlg</string>
  </property>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>932</x>
     <y>728</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>1062</x>
     <y>728</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>1165</width>
     <height>621</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabBarAutoHide">
    <bool>true</bool>
   </property>
   <property name="tabs" stdset="0">
    <string notr="true">sub</string>
   </property>
   <property name="pane_border" stdset="0">
    <string notr="true">border_all</string>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>生化</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableView" name="tableView_ch">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>200</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>44</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>免疫</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableView" name="tableView_im">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="scrolls" stdset="0">
        <string notr="true">layout_noborder</string>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>200</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>44</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>ISE</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableView" name="tableView_ise">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>200</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>44</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>shared/basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>AssayShiledDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1072</x>
     <y>719</y>
    </hint>
    <hint type="destinationlabel">
     <x>1066</x>
     <y>601</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
